docs/ops/detection/RegionYolo_1.md

```diff
@@ -6,7 +6,7 @@
 
 **Short description**: *RegionYolo* computes the coordinates of regions with probability for each class.
 
-**Detailed description**: This operation is directly mapped to the original YOLO layer. [Reference](https://arxiv.org/pdf/1612.08242.pdf)
+**Detailed description**: This operation is directly mapped to the [original YOLO layer](https://arxiv.org/pdf/1612.08242.pdf). 
```

Comment:
The detailed description should have the description of the algorithm. Also, there was an interesting feature of the RegionYolo implementation that works differently from the paper. I think that @mitruska did the analysis and can provide more information.

Reply:
@lazarevevgeny I spoke with @mitruska and she worked on different operator *ReorgYolo* (which also only link to paper in detail description). This operator was reviewed and closed in this form long time ago and I wanted to only add simple fixes like *Types* section. Do you really expect of me to learn internals of this complex operation and write *Detailed description* section ? :)

Reply:
Merged